openlibrary/plugins/importapi/code.py

```diff
@@ -459,7 +458,11 @@ def login(self, authstring):
         if not authstring:
             return
         authstring = authstring.replace("Basic ","")
-        username, password = base64.decodestring(authstring).split(':')
+        try:
+            username, password = base64.decodebytes(bytes(authstring, 'utf-8')).\
+                encode('utf-8').split(':')
+        except AttributeError:
+            username, password = base64.decodestring(authstring).split(':')
```

Comment:
Let's refactor for understandability...  ( please also repeat on line 609 )
```suggestion
        try:
            authstring = base64.decodebytes(bytes(authstring, 'utf-8')).encode('utf-8')
        except AttributeError:
            authstring = base64.decodestring(authstring)
        username, password = authstring.split(':')
```